go/oasis-node/cmd/debug/txsource/workload/transfer.go

```diff
@@ -0,0 +1,129 @@
+package workload
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+
+	"google.golang.org/grpc"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+	memorySigner "github.com/oasislabs/oasis-core/go/common/crypto/signature/signers/memory"
+	"github.com/oasislabs/oasis-core/go/common/logging"
+	"github.com/oasislabs/oasis-core/go/common/quantity"
+	consensus "github.com/oasislabs/oasis-core/go/consensus/api"
+	"github.com/oasislabs/oasis-core/go/consensus/api/transaction"
+	runtimeClient "github.com/oasislabs/oasis-core/go/runtime/client/api"
+	staking "github.com/oasislabs/oasis-core/go/staking/api"
+)
+
+const (
+	NameTransfer        = "transfer"
+	TransferNumAccounts = 10
+	TransferAmount      = 1
+)
+
+var logger = logging.GetLogger("cmd/txsource/workload/transfer")
+
+type transfer struct{}
+
+func (transfer) Run(gracefulExit context.Context, rng *rand.Rand, conn *grpc.ClientConn, cnsc consensus.ClientBackend, rtc runtimeClient.RuntimeClient) error {
+	// Load all the keys up front. Like, how annoyed would you be if down the line one of them turned out to be
+	// corrupted or something, ya know?
+	accounts := make([]struct {
+		signer          signature.Signer
+		reckonedNonce   uint64
+		reckonedBalance quantity.Quantity
+	}, TransferNumAccounts)
+	var err error
+	fac := memorySigner.NewFactory()
+	for i := range accounts {
+		accounts[i].signer, err = fac.Generate(signature.SignerEntity, rng)
+		if err != nil {
+			return fmt.Errorf("memory signer factory Generate account %d: %w", i, err)
+		}
+	}
+
+	// Read all the account info up front.
+	ctx := context.Background()
+	stakingClient := staking.NewStakingClient(conn)
+	for i := range accounts {
+		var account *staking.Account
+		account, err = stakingClient.AccountInfo(ctx, &staking.OwnerQuery{
+			Height: 0,
```

Comment:
Use `consensus.HeightLatest` instead of zero.

Reply:
ok